.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,5 @@
+### Expected behavior
+
+### Actual behavior
+
+### Steps to reproduce behavior
```

Comment:
We can add another block for the logs from slacker as well

Reply:
I like that :)